apps/android/app/src/main/java/chat/simplex/app/views/TerminalView.kt

```diff
@@ -31,18 +32,28 @@ import chat.simplex.app.views.helpers.*
 import com.google.accompanist.insets.ProvideWindowInsets
 import com.google.accompanist.insets.navigationBarsWithImePadding
 
+private var lastSuccessfulAuth: MutableState<Long?> = mutableStateOf(null)
+
 @Composable
 fun TerminalView(chatModel: ChatModel, close: () -> Unit) {
   val composeState = remember { mutableStateOf(ComposeState(useLinkPreviews = false)) }
-  BackHandler(onBack = close)
-  val authorized = remember { mutableStateOf(!chatModel.controller.appPrefs.performLA.get()) }
+  val lastSuccessfulAuth = remember { lastSuccessfulAuth }
+  BackHandler(onBack = {
+    lastSuccessfulAuth.value = null
+    close()
+  })
+  val authorized = remember { !chatModel.controller.appPrefs.performLA.get() }
   val context = LocalContext.current
-  LaunchedEffect(authorized.value) {
-    if (!authorized.value) {
-      runAuth(authorized = authorized, context)
+  LaunchedEffect(lastSuccessfulAuth) {
```

Comment:
lastSuccessfulAuth is a key, but it's changed inside runAuth... is it ok?

It crashed on me once - I sent log.